pkg/kncloudevents/message_sender.go

```diff
@@ -171,7 +171,7 @@ func RetryConfigFromDeliverySpec(spec duckv1.DeliverySpec) (RetryConfig, error)
 			}
 		case duckv1.BackoffPolicyLinear:
 			retryConfig.Backoff = func(attemptNum int, resp *nethttp.Response) time.Duration {
-				return delayDuration
+				return delayDuration * time.Duration(attemptNum)
```

Comment:
Do you need this? Can you do `attemptNum * delayDuration`?

Reply:
Doing that creates a mismatched type error: https://play.golang.org/p/X3A32Ew1Y_9